apps/dashboard/src/components/workflow-editor/steps/email/maily-config.tsx

```diff
@@ -235,4 +270,178 @@ export const createExtensions = (props: {
       },
     }),
   ];
+
+  if (isEnhancedDigestEnabled) {
+    extensions.push(
+      ButtonExtension.extend({
```

Comment:
the below piece of code allows us to hook into the node attributes update and set the `aliasFor` attribute that is used to resolve the variable